dlp/snippets/install-without-bom/pom.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.cloud</groupId>
+  <artifactId>dlp-install-without-bom</artifactId>
+  <packaging>jar</packaging>
+  <name>Google Cloud Data Loss Prevention Install Without Bom</name>
+  <url>https://github.com/googleapis/java-dlp</url>
```

Comment:
I think we need to change these?

Reply:
See example here: https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/dataproc/pom.xml#L4-L8

Reply:
Thanks for the catch. Done.